src/animal-quest.ts

```diff
@@ -13,7 +28,7 @@ export type AnimalQuest = {
   edition: string;
   description: string;
   broadcast: Date;
-  host: string;
+  host: HostName;
```

Comment:
```suggestion
  host: HostKey;
```
Instead of having the name here, now that we have a const/enum defining the hosts, we should reference it by key

Reply:
Ok! I implemented your suggestions and it's much shorter and clearer,  though with a slight change: made `HostKey` show actual hosts' name instead of just their key (would've have rendered just "maya" on the page otherwise), and renamed it to `HostName`.

Reply:
We should use `HostKey`, and then update the page to do a lookup. Otherwise, you have the same problem you already have that there is no way to map the episode host back to further data about them...

Reply:
Ahh I see! I sent the changes you suggested. I'll implement the website changes when it's up.
Thank you for the patience and apologies for the chaos :)